tic_tac_toe/game.py

```diff
@@ -63,22 +66,30 @@ def _check_winning_combinations(board, player):
     Returns the player (winner) of any of the winning combinations is completed
     by given player, or None otherwise.
     """
-    pass
-
+    horizontals = [ tuple((row, col)                for col in range(3)) for row     in range(3) ]
+    verticals   = [ tuple((row, col)                for row in range(3)) for col     in range(3) ]
+    diagonals   = [ tuple((row, abs(row - col_aux)) for row in range(3)) for col_aux in [0,2]    ]
+    
+    winning_combs = horizontals + verticals + diagonals
+    
+    if any(_is_winning_combination(board, comb, player) for comb in winning_combs):
```

Comment:
Thank you! The functions any() and all() help us make our code very concise and clear on its purpose.